plugins/scaffolder-backend/src/service/router.ts

```diff
@@ -115,7 +115,7 @@ export async function createRouter(
         return;
```

Comment:
missed a `.send` above here - maybe you needed to search for `.send(`, not `res.send` in the project?